src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -85,6 +85,11 @@ static bool exclusive = false;
 #endif
 #define htonll(a) ntohll(a)
 
+#define RBD_NBD_SOCK_PREFIX "ceph-client"
```

Comment:
Actually this should be $cluster-$name. See config_opts.h how the default is defined:
```
OPTION(admin_socket, OPT_STR, "$run_dir/$cluster-$name.asok") // default changed by common_preinit()
```